### ISSUE TITLE
Verschiedenes

### ISSUE THREAD

**Issue by AurelioSilver** [Xaratas/pillarsofeternity-german-patch-with-expansions#23]
Sammeissue für verschiedene Änderungsvorschlage.

Rassengenauigkeitsbonus vs Volkgenauigkeitsbonus

Wir haben Rasse, bei Obsidian heisst es Volk. Welche Variante ziehen wir vor?

**Comment by AurelioSilver** [comment]
Animat vs Animate

Wurde von den deutschen Übersetzern in Animate geändern. Übernehmen?

**Comment by Xaratas** [comment]
Der Genauigkeitsbonus ist ja gegen Tiere, Geister, Untote usw. Volk gilt im allgemeinen nur für Menschen, ergo ist Volk falsch.

Ich würd bei Animat bleiben. Zum einen aus Bestandsschutz und zum anderen gibt es sonst unschöne Stellen bei Animat-Rüstung z.B.

Wobei da die Bindestriche eigentlich auch weg können, alle beide.

```
data/localized/de_patch/text/quests/10_od_nua/10_qst_masters_tools.stringtabl
:      <DefaultText>Bei einem der inaktiven Wächter in den Ruinen habe ich ein Stück Adra-besetzter Animat-Rüstung gefunden.</DefaultText>
```

**Comment by AurelioSilver** [comment]
Darum hatte ich es gelassen. War aber der Annahme, es gibt den Bonus, wenn man ein Bestimmte Rasse spielt. :)

Für die Zusammenführung von Worten, sollten wir eine Liste anfertigen...

**Comment by AurelioSilver** [comment]
Corrosion Regen -> Säure-Regeneration

Müßte das nicht Zersetzungsregeneration heissen... wenn wir Corrosion mit Zersetzung übersetzen...

**Comment by Xaratas** [comment]
Schon, aber das ist nen komisches Wort. Wie wärs mit Hämerit? ^^
Hm, mal überlegen ob mir was besseres in den Sinn kommt.

**Comment by AurelioSilver** [comment]
Naja,,.Corrode (damage) => Zersetzungs(schaden)
Im deutschen "Original" heisst es Säureschaden
Von mir aus kann das auch weiterhin Säure-Regneration heissen. Fiel mir nur auf. Ist halt etwas inkonsitent.

**Comment by Xaratas** [comment]
Wir sollten schon in unseren Schadensklassen bleiben, nur was ist Zersetzungsheilung? Aufladen? Kleben? Kitten?

**Comment by AurelioSilver** [comment]
Biorepair... :) Aus der Zahnpastawerbung...

**Comment by AurelioSilver** [comment]
conversations\15_yenwood\15_si_yenwood

ID 202

Nach wenigen Augenblicken hebt der Marschall ein Horn, auf dem ein mächtiger Eber prangt, an seine Lippen. Er blickt dich erwartungsvoll an.

Nach wenigen Augenblicken hebt der Marschall ein Horn in der Form eines mächtigen Ebers an seine Lippen. Er blickt dich erwartungsvoll an.

Within a few moments, the marshal raises a brass horn in the visage of a mighty boar to his lips. He looks at you expectantly.

Ich hatte es bei der 2. Variante gelassen.

**Comment by Xaratas** [comment]
Die 2. Variante wär auch meine Wahl

**Comment by AurelioSilver** [comment]
eng: Lord's Authority
3.00 Autorität des Fürsten
3.01  Fürstliche Autorität

eng: Bravado
3.00 Wagemut
3.01 Bravado

Keine Meinung zu.

**Comment by Xaratas** [comment]
3.01 und 3.00

**Comment by AurelioSilver** [comment]
conversations\02_defiance_bay_first_fires\02_cv_duc_meeting
<ID>36</ID>
„Du denkst, irgendein verfaulender Leichnam oder eine freiwillige Wache könnte sich mir in den Weg stellen? Ich habe meine Antworten erhalten.“
"You think some rotting corpse or volunteer guard could stand in my way? I earned my answers."

<ID>37</ID>
„Sehen Sie mir in das Auge und nennen Sie mich einen Lügner, Euer Gnaden.“
"Look me in the eye and tell me I'm a liar, Your Grace."

<ID>38</ID>
„Mein Wort ist mein Eid. Ich fordere jeden auf, jemanden zu finden, der etwas anderes weiß.“
"My word is my oath. I challenge anyone here to find someone who knows different."

<ID>39</ID>
„Hören Sie sich um. Wenn ich Probleme erkenne, dann nehme ich die Angelegenheit in die eigene Hand.“
"Ask around. When I see problems I take matters into my own hands."

Bin darüber gestolpert... Vorschlag:

1) „Du denkst, irgendein verfaulender Leichnam oder eine freiwillige Wache könnte sich mir in den Weg stellen? Ich habe mir meine Antworten verdient.“
2) „Sehen Sie mir ins Gesicht und nennen Sie mich einen Lügner, Euer Gnaden.“
3) „Mein Wort ist mein Eid. Ich fordere jeden heraus, jemanden zu finden, der etwas anderes behauptet.“
4) „Hören Sie sich um. Wenn ich Probleme erkenne, dann nehme ich die Angelegenheit selbst in die Hand.“

zu 1) freiwillige Wache gefällt mir nicht so sehr. Aushilfswache? Freizeitwache?
zu 2) Das war viel zu wörtlich übersetzt. Scheint eine Redewendung zu sein.
Zu 3) Nicht sicher, ob es passt.

**Comment by Xaratas** [comment]
freiwillige Wache = Bürgerwehr
37: Sehen Sie mir in die Augen und bezichtigen Sie mich der Lüge, Euer Gnaden.
38: Ich schwöre die Wahrheit zu sagen. Möchte jemand der Anwesenden etwas anderes behaupten?

**Comment by AurelioSilver** [comment]
02_cv_leaden_key_agent
Als du dich dem Körper der Toten näherst, greifen dich wütende Gefühle an.
As you approach the woman's body, feelings of rage assault you.

Hm, ich hätte das genommen:
Als du dich dem Körper der Toten näherst, greifen dich Gefühle der Wut an.

**Comment by Xaratas** [comment]
Das 2. ist ja auch korrekt. Wobei „überwältigen“ statt „angreifen“ noch einen Tick besser ist.

**Comment by AurelioSilver** [comment]
55
Crumpled at your feet lies the body of a woman. She wears a dark robe that bears an insignia resembling a key.

As you approach the woman's body, feelings of rage assault you. The dead woman's voice howls in your ears, and her lingering consciousness rends at yours.

Vor deinen Füßen liegt am Boden der zerschlagene Körper einer Frau. Sie trägt eine schwarze Robe mit einem Abzeichen, das einem Schlüssel ähnelt.

Als du dich dem Körper der Toten näherst, greifen dich wütende Gefühle an. Die Stimme der Toten heult in deinen Ohren, und ihr nachklingendes Bewusstsein zerreißt dein eigenes.

Also so?

Als du dich dem Körper der Toten näherst, überwältigen dich Gefühle der Wut.

**Comment by Xaratas** [comment]
Ja, und wenn wir schon dabei sind. das „am Boden“ kann auf jeden Fall weg, das stört nur den Fluss. Der letzte Satz ist schon schwieriger. Ich glaub jetzt hab ich was, „dennoch“ oder „trotzdessen“? Sie ist auf jeden Fall tot.

Vor deinen Füßen liegt der zerschlagene Körper einer Frau. Sie trägt eine schwarze Robe mit einem angesteckten Abzeichen, das aussieht, wie ein grober Schlüssel.
Als du dich dem Körper der Toten näherst überwältigen dich Gefühle der Wut. Die Stimme der Toten heult in deinen Ohren und ihre dennoch verbliebene Seele zerreißt dir gewaltsam dein Bewusstsein.

**Comment by AurelioSilver** [comment]
Hm, hab Teile übernommen. Mit dem letzten Satz nicht bin ich nicht so glücklich. Seele wäre zuviel gesagt.

**Comment by Xaratas** [comment]
Naja, aber was haben wir denn Ingame? Haufenweise Seelen die nicht gehen wollen, weil sie noch nicht fertig sind. Aber an den anderen Stellen wird da glaube nicht von rumwabernden Bewusstsein gesprochen.

**Comment by Xaratas** [comment]
data_expansion2/…/abilities.stringtable
Aegis gegen Ägis austauschen? https://de.wikipedia.org/wiki/Aigis

**Comment by AurelioSilver** [comment]
Ja, kann man machen. Ich beweifle nur, dass die meisten das eine oder andere Wort kennen. :)
